### ISSUE TITLE
sensor.outdoorbell_call_state still Ringing

### ISSUE THREAD

**Issue by massiant70** [pergolafabio/Hikvision-Addons#155]
### What happened?

DS-KV8113-WME1(B)
After flawless installation I got this sensor.outdoorbell_call_state always ringing. Never change to idle or else.
So the registry report changing state from so long hours before.

### Add-on

Hikvision Doorbell

### What version of the add-on are you running?

3.0.10

### Installation type

Home Assistant add-on

### Relevant log output

```Text
2023-10-11 20:47:44.932 | DEBUG    | config:load_mqtt_config:130 - Loading MQTT configuration from supervisor
2023-10-11 20:47:44.932 | DEBUG    | config:mqtt_config_from_supervisor:36 - Requesting MQTT service configuration to supervisor
loop[2] find 2 mac and 0 ip
```

### Relevant configuration

```YAML
Mosquitto broker version: 6.3.1
```

**Comment by pergolafabio** [comment]
I think you have an issue with the MQTT, its not connecting to it

It should output something like:

```
2023-10-12 11:05:13.295 | DEBUG    | config:load_mqtt_config:130 - Loading MQTT configuration from supervisor
2023-10-12 11:05:13.296 | DEBUG    | config:mqtt_config_from_supervisor:36 - Requesting MQTT service configuration to supervisor
2023-10-12 11:05:13.311 | INFO     | sdk.utils:loadSDK:44 - Using OS: Linux with architecture: x86_64
loop[2] find 2 mac and 0 ip
2023-10-12 11:05:13.657 | INFO     | doorbell:authenticate:76 - Connected to doorbell: DS-KD8003
2023-10-12 11:05:13.689 | INFO     | doorbell:authenticate:76 - Connected to doorbell: DS-KH9510
```

Seems yours is stuck to mqtt, so on next event , it cant update the sensor, so what you see is the last state of the event

**Comment by massiant70** [comment]
2023-10-12 13:48:14.968 | DEBUG    | config:load_mqtt_config:130 - Loading MQTT configuration from supervisor
2023-10-12 13:48:14.968 | DEBUG    | config:mqtt_config_from_supervisor:36 - Requesting MQTT service configuration to supervisor
2023-10-12 13:48:14.994 | INFO     | sdk.utils:loadSDK:44 - Using OS: Linux with architecture: x86_64
loop[2] find 2 mac and 0 ip
2023-10-12 13:48:15.202 | INFO     | doorbell:authenticate:76 - Connected to doorbell: Outdoorbell
2023-10-12 13:48:15.202 | INFO     | event:__init__:87 - Setting up event handler: Console stdout
2023-10-12 13:48:15.203 | INFO     | mqtt:__init__:95 - Setting up event handler: MQTT

I set back log level to info and it seems like yours.
Now that someone ring to my outdoor station the status changed.
But it seems always to change neverending.
![Screenshot_mqtt](https://github.com/pergolafabio/Hikvision-Addons/assets/76493902/8993afee-8533-47f0-80f1-8fa1b3ceb58e)

**Comment by pergolafabio** [comment]
yes, thats a normal bevaviour, the addon is downloading all past events, give it a few hours untill all old evenst are cleared on your device , sometimes it takes an hour or even hours

should be good later :-)
